src/Filter/ChoiceFilter.php

```diff
@@ -35,29 +35,57 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
                 return;
             }
 
+            $isNullSelected = \in_array(null, $data['value'], true);
+            $data['value'] = array_filter($data['value'], static function ($value) {
+                return null !== $value;
+            });
```

Comment:
I think that here you can get empty array.

Reply:
Yes, but this is not a problem.